### ISSUE TITLE
Testing 3.3.6 - in Big Sur

### ISSUE THREAD

**Issue by cneidle** [DCS-LCSR/SignStream3#407]
I opened a file, and immediately got this error message.  I was, however, able to open the video file by choosing Edit Macro Unit from the menu.  I might also mention that "Resouce" is misspelled.

<img width="643" alt="Screen Shot 2021-02-26 at 2 38 17 PM" src="https://user-images.githubusercontent.com/13629122/109347023-80dae000-7840-11eb-879d-b4720e8deb33.png">

**Comment by cneidle** [comment]
Problems that appear to be fixed:  color issues; problems with imposing constraints on overlap;  to include non-dominant hand of 2-handed signs; problem with failure of non-dom hand sign type to appear when defined.

The zoom seems to be working better.  And when the current frame is the last frame in the utterance, what remains in view is now appropriate.

Selection of sign types seems to work better, although I encountered one error when I tried to convert a lexical sign to a loan sign; but was unable to reproduce it in order to report it.
